x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1192,6 +1192,34 @@ var security = (function () {
         target: "",
     });
 
+    var renameNewProcessFields = new processor.Chain()
+        .Convert({
+            fields: [
+                {from: "winlog.event_data.NewProcessId", to: "process.pid", type: "long"},
+                {from: "winlog.event_data.NewProcessName", to: "process.executable"},
+		{from: "winlog.event_data.ParentProcessName", to: "process.parent.name"},
+            ],
+            mode: "rename",
+            ignore_missing: true,
+            fail_on_error: false,
+        })
+        .Add(function(evt) {
+            var name = evt.Get("process.name");
+            if (name) {
+                return;
+            }
+            var exe = evt.Get("process.executable");
+            evt.Put("process.name", path.basename(exe));
+        })
+	.Add(function(evt) {
+	    var cl = evt.Get("winlog.event_data.CommandLine");
+	    if (!cl) {
+		return;
+	    }
+	    evt.Put("process.args", cl.match(/\S+/g));
```

Comment:
There's a windows function for this that we have exposed as a module. For example see:

https://github.com/elastic/beats/blob/ca81776033889ad59bc3b571e58ea1a90fba4f5e/x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js#L245

`process.command_line` is a new field that is being added to ECS (https://github.com/elastic/ecs/pull/599/files#diff-79c5c20c38369780c00142b4bc7bb519R2980). It will contain the raw command line string.